taskcluster/ci/config.yml

```diff
@@ -10,7 +10,7 @@ taskgraph:
     repositories:
         mobile:
             name: "Firefox for iOS"
-    cached-task-prefix: mobile.firefox-ios
+    cached-task-prefix: mobile.v2.firefox-ios
```

Comment:
I really like the new index routes and the fact that we're standardizing everywhere 👍